frontend/packages/kubevirt-plugin/integration-tests/v2v/vmware/v2v.vmware.import.negative.scenario.incorrect.names.ts

```diff
@@ -0,0 +1,109 @@
+import { asyncForEach, click } from '@console/shared/src/test-utils/utils';
+import { VirtualMachineModel } from '@console/kubevirt-plugin/src/models';
+import { VmwareImportWizard } from '../../tests/models/vmwareImportWizard';
+import { vmwareVMConfig } from './v2v.configs';
+import * as view from '../../views/importWizard.view';
+import { wrongValues } from '../../tests/utils/constants/vm';
+import { clickKebabAction } from '../../views/wizard.view';
+import { NetworkInterfaceDialog } from '../../tests/dialogs/networkInterfaceDialog';
+import { browser } from 'protractor';
+import { DiskDialog } from '../../tests/dialogs/diskDialog';
+
+const {
+  provider,
+  instanceConfig,
+  name,
+  sourceVMName,
+  operatingSystem,
+  flavorConfig,
+  workloadProfile,
+} = vmwareVMConfig;
+
+describe('VMWare Wizard validation, negative tests', () => {
+  const wizard = new VmwareImportWizard();
+
+  beforeEach(async () => {
+    await wizard.openWizard(VirtualMachineModel);
+  });
+
+  afterEach(async () => {
+    await wizard.closeWizard();
+  });
+
+  async function processInstanceConnection() {
+    await wizard.selectProvider(provider);
+    await wizard.waitForSpinner();
+    await wizard.configureInstance(instanceConfig);
+
+    await wizard.connectToInstance();
+    await wizard.waitForSpinner();
+
+    await wizard.selectSourceVirtualMachine(sourceVMName);
+    await wizard.waitForSpinner();
+    await click(view.nextButton);
+
+    if (operatingSystem) {
+      await wizard.selectOperatingSystem(operatingSystem as string);
+    }
+    if (flavorConfig) {
+      await wizard.selectFlavor(flavorConfig);
+    }
+    if (workloadProfile) {
+      await wizard.selectWorkloadProfile(workloadProfile);
+    }
+  }
+
+  it('VMWare - Import Wizard shows warning when using incorrect VM name', async () => {
+    await processInstanceConnection();
+    await asyncForEach(wrongValues, async (curValue) => {
+      const err = await wizard.fillName(curValue);
+      expect(err).toBeDefined();
+    });
+  });
+
+  it('VMWare - Import Wizard shows warning when using incorrect VM NIC name', async () => {
+    await processInstanceConnection();
+    await wizard.fillName(name);
+    await wizard.next();
+    const importedNICs = await wizard.getImportedNics();
+    const NICDialog = new NetworkInterfaceDialog();
+    if (importedNICs.length > 0) {
+      const nic = importedNICs[0];
+      await clickKebabAction(nic.name, 'Edit');
+      await asyncForEach(wrongValues, async (curValue) => {
+        const err = await NICDialog.fillName(curValue);
+        expect(err).toBeDefined();
+      });
+      await browser.sleep(10000);
+      await click(view.modalCancelButton);
+      await browser.sleep(10000);
+    } else {
+      pending('No network interfaces found');
```

Comment:
can you please point to where is pending defined?

Reply:
https://jasmine.github.io/api/2.9/global.html#pending

Reply:
thanks